tools/ldb_cmd.cc

```diff
@@ -3425,10 +3429,49 @@ void ListFileRangeDeletesCommand::DoCommand() {
     TEST_SYNC_POINT_CALLBACK(
         "ListFileRangeDeletesCommand::DoCommand:BeforePrint", &out_str);
     fprintf(stdout, "%s\n", out_str.c_str());
+  }
+}
+
+void UnsafeRemoveSstFileCommand::Help(std::string& ret) {
+  ret.append("  ");
+  ret.append(UnsafeRemoveSstFileCommand::Name());
+  ret.append(" <sst_file_name> ");
+  ret.append("\n");
+}
+
+UnsafeRemoveSstFileCommand::UnsafeRemoveSstFileCommand(
+    const std::vector<std::string>& params,
+    const std::map<std::string, std::string>& options,
+    const std::vector<std::string>& flags)
+    : LDBCommand(options, flags, false /* is_read_only */,
+                 BuildCmdLineOptions({})) {
+  if (params.size() != 1) {
+    exec_state_ = LDBCommandExecuteResult::Failed("SST file must be specified");
+  } else {
+    file_ = params.at(0);
+  }
+}
+
+void UnsafeRemoveSstFileCommand::DoCommand() {
+  if (!db_) {
+    assert(GetExecuteState().IsFailed());
+    return;
+  }
+
+  Status st = db_->DeleteFile(file_, true);
```

Comment:
Just to understand the logic. You mentioned that the DB might fail to restart when an SST file is corrupted. Then how can we guarantee that we can open the DB and get `db_`?

Reply:
set `opt.paranoid_checks = false`